### PR TITLE
calibre-web: 0.6.11 -> 0.6.12

### PR DESCRIPTION
**Pull request by ambroisie**
###### Motivation for this change

The current version fails to build, bumping the version updates the dependency requirements such that it builds now.

I am not sure about the packaging of PyPDF3, as it is mostly just a copy-paste of PyPDF2, notably I'm not sure why `glibclocales` is needed as a `buildInput`.

###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [X] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Relase notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 128265` at 6e29c6a6 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built successfully:</summary>
  <ul>
    <li>calibre-web</li>
    <li>python38Packages.pypdf3</li>
    <li>python39Packages.pypdf3</li>
  </ul>
</details>
<details>
  <summary>2 suggestions:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/stale-substitute.md">stale-substitute</a>

Stale substituteInPlace detected.
substituteStream(): WARNING: pattern 'Babel>=1.3, <2.9' doesn't match anything in file 'setup.cfg'
Near pkgs/servers/calibre-web/default.nix:68:0:
```
   |
68 |     description = "Web app for browsing, reading and downloading eBooks stored in a Calibre database";
   | ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/stale-substitute.md">stale-substitute</a>

Stale substituteInPlace detected.
substituteStream(): WARNING: pattern 'unidecode>=0.04.19,<1.2.0' doesn't match anything in file 'setup.cfg'
Near pkgs/servers/calibre-web/default.nix:68:0:
```
   |
68 |     description = "Web app for browsing, reading and downloading eBooks stored in a Calibre database";
   | ^
```
</li>
  </ul>
</details>

File: pkgs/development/python-modules/pypdf3/default.nix [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  ];
+
+  meta = with lib; {
+    description = "A Pure-Python library built as a PDF toolkit";
+    homepage = "https://github.com/sfneal/PyPDF3";
+    license = licenses.bsd3;
+    maintainers = with maintainers; [ ambroisie ];
+  };
+
+}
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/pypdf3/default.nix:38]:
```suggestion
  };
}
```

File: pkgs/development/python-modules/pypdf3/default.nix [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+  LC_ALL = "en_US.UTF-8";
+  buildInputs = [ glibcLocales ];
+
+  checkPhase = ''
+    ${python3.interpreter} -m unittest tests/*.py
+  '';
+
+  doCheck = true;
+
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/pypdf3/default.nix:26]:
```suggestion
```

File: pkgs/development/python-modules/pypdf3/default.nix [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-DGKpR4p3z8tw4gKi5Hmj09svysD3Hkn4NklhgROmEAU=";
+  };
+
+  LC_ALL = "en_US.UTF-8";
+  buildInputs = [ glibcLocales ];
+
+  checkPhase = ''
+    ${python3.interpreter} -m unittest tests/*.py
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/pypdf3/default.nix:22]:
```suggestion
    ${python.interpreter} -m unittest tests/*.py
```

File: pkgs/development/python-modules/pypdf3/default.nix [hunk 4]
```diff
@@ -0,0 +1,38 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, glibcLocales
+, python3
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/pypdf3/default.nix:5]:
```suggestion
, python
```

File: pkgs/development/python-modules/pypdf3/default.nix [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+, tqdm
+}:
+
+buildPythonPackage rec {
+  pname = "PyPDF3";
+  version = "1.0.5";
+
+  src = fetchPypi {
+    inherit pname version;
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/pypdf3/default.nix:14]:
```suggestion
  pname = "pypdf3";
  version = "1.0.5";

  src = fetchPypi {
  	pname = "PyPDF3";
    inherit version;
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Comment by ambroisie** [comment]
Addressed your suggestions and the stale substitutes from `nixpkgs-review`.

**Pull request merged**